client/packages/system/src/Item/Components/PackSize/PackSizeUnitNameCell.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { ReactElement } from 'react';
+import {
+  RecordWithId,
+  CellProps,
+  BasicCellLayout,
+} from '@openmsupply-client/common';
+
+// We don't have pack variants here currently, so we just display the unit name
+export const PackSizeUnitNameCell =
```

Comment:
Maybe worth getting rid of this one? Just render usual Cell, accessor `({ rowData }) => rowData.unitName` ?